benchmarks/dynamo/common.py

```diff
@@ -991,7 +991,7 @@ def maybe_mark_profile(*args, **kwargs):
                 model, example_inputs, args.devices[0]
             )
         else:
-            frozen_model_iter_fn = torch._dynamo.run(model_iter_fn)
+            frozen_model_iter_fn = optimize_ctx(model_iter_fn)
```

Comment:
This looks fine, but can you elaborate more on why you need to make the change?

Reply:
Hi @desertfire, 
Sorry for late response on that. In general, `speedup_experiment` calculates speedup over eager. Imo, it should take defined optimizations in account like e.g. compile backend. This will not happen here if we do not take `optimize_ctx`. It is made similarly for other performance metrics, e.g. `dynamo_latency, dynamo_peak_mem` calculated in `performace_test` method - we take there `optimize_ctx`.